src/controllers/addEventController.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express'
+import querySchema from './../validation/addEventValidate'
+import { Event } from './../models'
+
+const addEvent = async (req: Request, res: Response, next: NextFunction) => {
+  const data = req.body
+  try {
+    await querySchema.validateAsync(req.body)
+    const event = await Event.create({
+      name: data.name,
+      description: data.description,
+      img: data.img,
+      status: data.status,
+      startTime: data.startTime,
+      endTime: data.endTime,
+      longitude: data.longitude,
+      latitude: data.latitude
+    })
+
+    res.status(200).json({
+      message: 'inserted successfully'
+    })
```

Comment:
you don't have to pass the 200 status code because it's the default one

it's very good to return a success message
but it's more good if you return the created event with it
```suggestion
    res.json({
      message: 'inserted successfully',
      data: event
    })
```

Reply:
and a more enhancement is to make our messages available globally
 to make them more consistent and readable
 
 you can make it with enums 
 and you can use it like this
 ```suggestion
    res.status(200).json({
      message: Messages.SUCCESS
    })
```